### PR TITLE
Fix to handle when a pseudoLocales type is object

### PR DESCRIPTION
**Pull request by gouniLee**
* In order to work correctly with loctool change: https://github.com/iLib-js/loctool/pull/165
* plugin should handle `project.psseudoLocales` and type is  `object` always. not `array`

### CODE REVIEW

File: JavaScriptFileType.js [hunk 1]
```diff
@@ -39,17 +39,15 @@ var JavaScriptFileType = function(project) {
...
-    if (typeof project.pseudoLocale === "string") {
-        project.pseudoLocale = [project.pseudoLocale];
-    }
-
     // generate all the pseudo bundles we'll need
-    project.pseudoLocale && project.pseudoLocale.forEach(function(locale) {
-        var pseudo = this.API.getPseudoBundle(locale, this, project);
-        if (pseudo) {
-            this.pseudos[locale] = pseudo;
+    if (project.pseudoLocales && typeof project.pseudoLocales == 'object'){
```
Comment (ehoogerbeets) [inline JavaScriptFileType.js]: Need to be able to deal with the situation where it is an array OR an object.

Comment (gouniLee) [reply JavaScriptFileType.js]: looks like it never happens to return the `array` type. isn't it?
https://github.com/iLib-js/loctool/blob/nopseudo/lib/Project.js#L86
https://github.com/iLib-js/loctool/blob/nopseudo/lib/PseudoFactory.js#L227

Comment (ehoogerbeets) [reply JavaScriptFileType.js]: In the "nopseudo" branch, then yes.
But in an older version of loctool, then no

Comment (gouniLee) [reply JavaScriptFileType.js]: I updated to support both cases and check if these are correctly generating pseudo translation data

**Review by ehoogerbeets — APPROVED** [review]

**Pull request merged**